frontend/ui/src/components/kanban-board/task-card.tsx

```
@@ -52,151 +58,250 @@
+  const { handleDrawerOpen } = useKanbanDrawer()
   return (
     <div
       ref={outerRef}
+      onClick={() => handleDrawerOpen(card.id.toString())}
+      onKeyDown={(e) => {
+        if (e.key === "Enter" || e.key === "Space") {
+          e.preventDefault()
+          handleDrawerOpen(card.id.toString())
+        }
+      }}
+      role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)